### DISCUSSION TITLE
MRF Validator - error NDC Code Type requires elements not required until 1/1/25

### DISCUSSION THREAD

**Discussion by BBDURMC** [CMSgov/hospital-price-transparency#72] [Q&A] [answered]
The MRF validator is failing on NDC Code Type for missing units and drug type. However, these data elements are not required on 7/1/24.  Please correct the validator to remove these requirements.

Thank you.

**Comment by CarlsonErin** [comment, marked as answer]
If an MRF does not conform to the hospital price transparency final rule form and manner requirements, the Online Validator will generate an output consisting of 'errors' and 'warnings'. Online Validator **'errors'** represent requirements that are enforced **beginning July 1, 2024**, whereas **'warnings'** represent requirements that are **enforced beginning January 1, 2025**.

Errors "fail" the validator.  Warnings provide notice of the possibility of a future failure beginning January 1, 2025.

**Comment by zprice22** [comment]
Question also, the conditional requirements indicate, "If code type is NDC, then the corresponding drug unit of measure and drug type of measure data element must be encoded. Required beginning 1/1/2025.". Is this indicating that the unit of measure and drug type are only required if the code type of NDC is utilized? What if HCPCS is utilized and not the NDC? Will this not error?

**Reply by RidenourK** [reply]
@zprice22. Thank you for your question. 

The conditional requirements must be met or the MRF uploaded to the Validator will produce warnings and/or error messages. Leveraging the Validator helps ensure adherence to CMS form and manner requirements (45 CFR 180.50(c)(2)), but it does not certify that your MRF fully complies with all HPT requirements at 45 CFR 180.50. 

Beginning January 1, 2025, each hospital must encode in its machine-readable file the drug unit and type of measurement for all drugs.

**Comment by CarlsonErin** [comment]
Works for me.

From: RidenourK ***@***.***>
Sent: Friday, April 12, 2024 11:58 AM
To: CMSgov/hospital-price-transparency ***@***.***>
Cc: Erin C Carlson ***@***.***>; Comment ***@***.***>
Subject: [EXT] Re: [CMSgov/hospital-price-transparency] MRF Validator - error NDC Code Type requires elements not required until 1/1/25 (Discussion #72)

@zprice22. Thank you for your question. The conditional requirements must be met or the MRF uploaded to the Validator will produce warnings and/or error messages. Leveraging the Validator helps ensure adherence to CMS form and manner requirements


@zprice22<https://github.com/zprice22>. Thank you for your question.

The conditional requirements must be met or the MRF uploaded to the Validator will produce warnings and/or error messages. Leveraging the Validator helps ensure adherence to CMS form and manner requirements (45 CFR 180.50(c)(2)), but it does not certify that your MRF fully complies with all HPT requirements at 45 CFR 180.50.

Beginning January 1, 2025, each hospital must encode in its machine-readable file the drug unit and type of measurement for all drugs.
